src/app/dataScripts/fetchSpendingOverTimeData.ts

```diff
@@ -112,15 +112,17 @@ function cleanDataDump(
     const perCapitaSpendingOverMonths: MonthlySpendingOverTime = [];
     dataDump.results.reduce(
         (sum, { aggregated_amount, time_period: { fiscal_year, month } }) => {
-            const aggregatedPerCapita = sum + aggregated_amount / population;
+            const aggregatedTotal = sum + aggregated_amount;
+            const aggregatedPerCapita = (sum + aggregated_amount) / population;
```

Comment:
```suggestion
            const aggregatedPerCapita = aggregatedTotal / population;
```

Reply:
This is where I was seeing that bug before, but in retrospect, this was a misinterpretation of "It should also agree in amount with the spending tracker at the top of the page." from #104, and confusing that with the app before #96. It did gets fixed with that, so no worries.